test/system/admin/instructors/instructors_edit_test.rb

```diff
@@ -28,7 +28,11 @@ class InstructorsEditTest < ApplicationSystemTestCase
     visit edit_admin_instructor_path(@instructor)
 
     within 'form trix-editor' do
-      assert_text @instructor.resume.to_plain_text
+      lines = @instructor.resume.to_plain_text.lines.map(&:strip).compact_blank
+
+      lines.each do |line|
+        assert_text line
+      end
```

Comment:
To avoid repeat this code, create a custom assert. Something like this:

```ruby
def assert_rich_text(rich_text)
  rich_text.to_plain_text.lines.map(&:strip).compact_blank.each do |line|
    assert_text line, "Expected text line to appear: '#{line}'"
  end
end
```

```ruby
assert_rich_text(@instructor.resume)
```